packages/wrangler/src/__tests__/deployments.test.ts

```diff
@@ -71,7 +84,7 @@ describe("deployments", () => {
 	});
 
 	it("should log deployments for script with passed in name option", async () => {
-		await runWrangler("deployments --name somethingElse");
+		await runWrangler("deployments --name something-else");
```

Comment:
Was there a reason to change this deployment script name in this PR?

Reply:
Yes since in the previous test there is some verification for the toml parsing that the script name is all lowercase, I changed this one to lowercase for consistency

Reply:
But script names do not need to be lower-case, right??
Or did I miss that requirement...?

Reply:
I think they do, in the dash script names are automatically made lowercase.
Here is the check in wrangler
```javascript
(typeof value === "string" && /^$|^[a-z0-9_ ][a-z0-9-_ ]*$/.test(value)) ||
```